flocker/testtools/_base.py

```diff
@@ -143,8 +149,13 @@ def __init__(self, *args, **kwargs):
 
     def setUp(self):
         super(AsyncTestCase, self).setUp()
+        log.msg("--> Begin: %s <--" % (self.id()))
         self.useFixture(_SplitEliotLogs())
 
+    def tearDown(self):
+        super(AsyncTestCase, self).tearDown()
+        log.msg("--> End: %s <--" % (self.id()))
```

Comment:
Best to put these two the other way around (i.e. upcall last) so that there's symmetry between setUp & tearDown. 

Also for `TestCase`.